var/spack/repos/builtin/packages/vmc/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Vmc(CMakePackage):
+    """The Virtual Monte Carlo (VMC) library"""
+
+    homepage = "https://github.com/vmc-project/vmc"
+    git      = 'https://github.com/vmc-project/vmc.git'
+    url      = "https://github.com/vmc-project/vmc/archive/v1-0.tar.gz"
```

Comment:
```suggestion
    url      = "https://github.com/vmc-project/vmc/archive/v1-0-p2.tar.gz"
```
This doesn't affect fetching, but it does affect the statistics reported by `spack url summary`

Reply:
Fixed.